README.md

```diff
@@ -45,3 +116,29 @@ trigger the [scope
 inspector](https://code.visualstudio.com/api/language-extensions/syntax-highlight-guide#scope-inspector)
 from the Command Palette (`Ctrl+Shift+P`) with the `Developer: Inspect Editor
 Tokens and Scopes` command.
+
+## User Guide 
+
+After installing the extension, in order to utilise it you will need to load an ATF file. 
+If you do not have an ATF file, there are many freely available on the 
+[ORACC site archive](http://oracc.museum.upenn.edu/doc/search/index.html).
+
+<img src="media/generalUI.png" align="center">
+
+Once you have an ATF file to load, simple locate the File option on the menu bar at the top left of the window, and choose the Open File option from the drop down menu. This will bring up a window allowing you to locate the ATF file. Once located, select it and choose open.
+
+One key feature is the ability to display text in multiple different languages, and features viewing panels that can display the text from right-to-left (eg: Arabic).
+
+Once an ATF file has been loaded, you should have two extra icons on the top right.
+
+<img src="media/buttons.png" align="center">
+
+The first of these in the Show Arabic Preview button, which brings up a secondary panel displaying the correct orientation of the Arabic text (right to left). 
+
+<img src="media/arabicpreview.png" align="center">
```

Comment:
The file is named `arabicpreview.PNG` (with a capitalised extension) and so won't be displayed on some systems. I suggest renaming it to `arabicpreview.png` to match the other files, and the name here (same for `validateATF.PNG`)